cmake/scripts/buildinfo.cmake

```diff
@@ -61,7 +61,7 @@ if (SOL_COMMIT_HASH AND SOL_LOCAL_CHANGES)
 endif()
 
 set(SOL_VERSION_COMMIT "commit.${SOL_COMMIT_HASH}")
-set(SOl_VERSION_PLATFORM ETH_BUILD_PLATFORM)
+set(SOL_VERSION_PLATFORM ETH_BUILD_PLATFORM)
```

Comment:
What changes did you make here?

Reply:
Looks like a typo: `L` was not capitalized before. Apparently we never use it so no one ever noticed.

This info should be in the description/title though.

Reply:
Oh, now I see it! It wasn’t obvious to me at first. I looked at it too quickly hahaha.